about.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+    <link rel="stylesheet" href="abtt.css">
+    <title>Bole Festival 2023 About page</title>
+</head>
+
+<body>
+    <header>
+        <section class="header" id="header">
+            <nav class="nav-container1" id="nav-container1">
+                <div class="nav-menu1" id="nav-menu1">
+                    <ul class="nav-list1" id="">
+                        <li class="nav-item1"><a href="#">
+                                <i class="ri-facebook-fill"></i>
+                            </a></li>
+                        <li class="nav-item1"><a href="#">
+                                <i class="ri-twitter-fill"></i>
+                            </a></li>
+                        <li class="nav-item1"><a href="#">Register</a></li>
+                        <li class="nav-item1"><a href="#">Connect</a></li>
+                        <li class="nav-item1"><a href="#">Donate</a></li>
+                    </ul>
+                </div>
+            </nav>
+            <nav class="nav-container2 flex-nav" id="nav-container2">
+                <div class="nav-toggle" id="nav-toggle">
+                    <i class="ri-menu-fill"></i>
+                </div>
+                <ul class="nav-list2" id="nav-list2">
```

Comment:
- [ ]  You Hamburg menu is working fine, but I recommend you to style it based on the provided design so that it relatively match your design. 
![image](https://user-images.githubusercontent.com/43877689/227660311-e77d0c34-f47f-40ec-b7ed-ca1ca3f87182.png)